src/Chunk.ts

```diff
@@ -778,7 +778,9 @@ export default class Chunk {
 					renderedResolution,
 					resolution instanceof Module &&
 						!(resolution.facadeChunk && resolution.facadeChunk.strictFacade) &&
-						resolution.namespace.exportName!,
+						!!resolution.namespace.exportName &&
+						// We only need one of the export names to bind to
+						resolution.namespace.exportName[0],
```

Comment:
@lukastaegert can you confirm that in this scenario its irrelevant which export name is used?

Reply:
As far as I'm aware - these names are more for usability than anything else as far as I recall.